src/components/StarRating/src/Star.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+	<!-- eslint-disable max-len -->
+	<svg
+		:class="$s.Star"
+		viewBox="0 0 16 16"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+		@mouseenter="$emit('hover')"
```

Comment:
Should we just label the emitted events 'mouseenter' and 'mouseleave' to map to the native events?

Reply:
I'd be fine with that or perhaps `on:hover` and `off:hover`